src/assets/i18n/pt-PT.json5

```diff
@@ -189,11 +189,14 @@
   "admin.registries.bitstream-formats.table.internal": "Interno",
 
   // "admin.registries.bitstream-formats.table.mimetype": "MIME Type",
-  "admin.registries.bitstream-formats.table.mimetype": "MIME Type",
+  "admin.registries.bitstream-formats.table.mimetype": "Tipo MIME",
 
   // "admin.registries.bitstream-formats.table.name": "Name",
   "admin.registries.bitstream-formats.table.name": "Nome",
 
+  // "admin.registries.bitstream-formats.table.selected": "Selected bitstream formats",
+  "admin.registries.bitstream-formats.table.selected": "Formatos de ficheiros selecionados",
```

Comment:
Portuguese from Portugal is "seleccionados" instead of "selecionados" which is the Portuguese from Brasil.